assets/sass/widgets/_googlesitekit-widget-searchFunnel.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Search Funnel styles.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-cta--analytics {
+		--cta-analytics-gap: #{$grid-gap-phone};
+
+		@media (min-width: $bp-tablet) {
+			--cta-analytics-gap: #{$grid-gap-desktop};
+		}
+
+		background-color: $c-analytics-cta-bg;
+		display: flex;
+		flex-wrap: wrap-reverse;
+		font-size: 14px;
+		gap: var(--cta-analytics-gap);
+		padding: var(--cta-analytics-gap);
+
+		.mdc-button {
+			display: block;
+			letter-spacing: normal;
+			margin-top: var(--cta-analytics-gap);
+			text-transform: none;
+			width: 100%;
+		}
+
+		.googlesitekit-cta--activate-analytics {
+			display: flex;
+			flex-wrap: wrap;
+			gap: var(--cta-analytics-gap);
+		}
+
+		.googlesitekit-cta--graph {
+			--cta-analytics-graph-gap: calc(var(--cta-analytics-gap) / 2);
+
+			@include shadow;
+
+			background-color: $c-base;
+			border-radius: 4px;
+			display: flex;
+			flex: 1;
+			flex-direction: column;
+			min-width: 120px;
+			padding: var(--cta-analytics-graph-gap);
+
+			.googlesitekit-cta--graph--title {
+				color: $c-secondary;
+				display: block;
```

Comment:
`display: block` is irrelevant here since the element used (`div` in this case or `p` or a heading tag as I suggested are already block elements)